### PR TITLE
Can now add and remove nodes from the zwave network.

### PR DESCRIPTION
**Pull request by dhylands**
r? @julienw

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -99,6 +99,14 @@ impl ZWaveManager {
         }
     }
 
+    pub fn add_node(&mut self, home_id: u32, secure: bool) -> Result<(), ()> {
+        self.ozw_manager.add_node(home_id, secure).and(Ok(()))
+    }
+
+    pub fn remove_node(&mut     self, home_id: u32) -> Result<(), ()> {
```
Comment (julienw) [inline src/lib.rs]: nit: seems like you have too many spaces here :)

File: src/lib.rs [hunk 2]
```diff
@@ -99,6 +99,14 @@ impl ZWaveManager {
         }
     }
 
+    pub fn add_node(&mut self, home_id: u32, secure: bool) -> Result<(), ()> {
```
Comment (julienw) [inline src/lib.rs]: the lib uses mutexes for this call, so we don't need a &mut, a shared reference is good enough.

File: src/lib.rs (hunk 1 above)
Comment (julienw) [inline src/lib.rs]: same remark about &mut here

**Comment by dhylands** [comment]
Removed mut

**Pull request merged**